### ISSUE TITLE
(Feature request) Add info on how to set TRILIUM_DATA_DIR on Windows

### ISSUE THREAD

**Issue by yuuurchyk** [TriliumNext/Trilium#4853]
### Describe feature

When reading the wiki [page](https://github.com/zadam/trilium/wiki/Data-directory), I've found out that you can specify the `TRILIUM_DATA_DIR` environment variable to provide a custom path to the data directory.

Although it's working on windows, the explicit mention of this operating system is missing in the **Changing the location of data directory** section of the corresponding wiki page.

Could you please add a brief mention of windows OS, since it may cause confusion that it may not be possible to do (especially for non-tech people). Sth like:

### Changing the location of data directory

**Windows**

Press start, search for `Edit the system environment variables`. In the popup window press on `Environment variables...`. Then press `New...` in **User variables section**. Fill in `Variable name:` field with `TRILIUM_DATA_DIR`, then press `Browse Directory...` and select the data folder of interest. Close all the popup windows by repeatedly pressing `OK`.

**Comment by LeoRya** [comment]
so did u solve it？ I still dont know how to change the dir.

**Comment by yuuurchyk** [comment]
@LeoRya yes, you just need to set the `TRILIUM_DATA_DIR` environment variable. I think your best bet would be to google how to do it on windows in case the explanation mentioned in the ticket's description is not clear
